api/Models/DB/Databasechangelog.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SS.Api.Models.DB
+{
+    public partial class Databasechangelog
+    {
+        public string Id { get; set; }
+        public string Author { get; set; }
+        public string Filename { get; set; }
+        public DateTime Dateexecuted { get; set; }
+        public int Orderexecuted { get; set; }
+        public string Exectype { get; set; }
+        public string Md5sum { get; set; }
+        public string Description { get; set; }
+        public string Comments { get; set; }
+        public string Tag { get; set; }
+        public string Liquibase { get; set; }
```

Comment:
Are we using liquibase for this project? Or is this to support existing sheriff scheduling data.

Reply:
This was just a scaffolding of the existing database using EF. I've removed this model but haven't pushed it to the PR.